lib/cassandrax/connection.ex

```diff
@@ -153,4 +153,8 @@ defmodule Cassandrax.Connection do
 
   defp do_intersperse_map([element | rest], separator, mapper),
     do: [mapper.(element), separator | do_intersperse_map(rest, separator, mapper)]
+
+  defp extract_order({:asc, field}), do: [quote_name(field) | " ASC"]
+  defp extract_order({:desc, field}), do: [quote_name(field) | " DESC"]
+  defp extract_order(field), do: quote_name(field)
```

Comment:
I think we can move these closer to where it's being used, as it is specific to `order_by` and not used by any other functions.

That aside, I believe using the reverse notation should work better:
```elixir
# In here
defp extract_order({field, :asc}), do: [quote_name(field) | " ASC"]

# When using `order_by`
...
|> order_by(foo: :asc, bar: :desc)
# which should output `ORDER BY "foo" ASC, "bar" DESC`
```
This allows you to keep in the final CQL the same order or fields you defined in your query.